dockerfiles/keycloak/kc_realm_user.sh

```diff
@@ -40,10 +40,14 @@ if [ "${CHE_SELF__SIGNED__CERT}" != "" ]; then
     /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/cli/add_openshift_certificate.cli && rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
 fi
 
+# POSTGRES_PORT is assigned by Kubernetes controller
+# and it isn't fit to docker-entrypoin.sh.
+unset POSTGRES_PORT
+
 echo "Starting Keycloak server..."
 
-/opt/jboss/keycloak/bin/standalone.sh -Dkeycloak.migration.action=import \
-                                      -Dkeycloak.migration.provider=dir \
-                                      -Dkeycloak.migration.strategy=IGNORE_EXISTING \
-                                      -Dkeycloak.migration.dir=/scripts/ \
-                                      -Djboss.bind.address=0.0.0.0
+exec /opt/jboss/docker-entrypoint.sh -Dkeycloak.migration.action=import \
```

Comment:
why the change of script here? what does docker-entrypoint.sh do that standalone.sh doesn't? (Also, aside, "container-entrypoint" would be better. :) )

Reply:
Please read [docker-entrypoint.sh](https://github.com/jboss-dockerfiles/keycloak/blob/502c05c7b58840c03d6c619b609b4257da5ac70c/server/tools/docker-entrypoint.sh) that stored in the base image.
Some important environment variables are parsed in there. `standalone.sh` is called by `docker-entrypoint.sh` after parsing.

It's best to ask maintainers of the `jboss-dockerfiles` organization (not me) if you don't like the name `docker-entrypoint.sh`.